package.json

```diff
@@ -63,6 +63,8 @@
   "dependencies": {
     "@elastic/elasticsearch": "7.9.0",
     "@elastic/elasticsearch-mock": "0.3.0",
+    "@material-ui/core": "4.11.0",
```

Comment:
You can remove this, we have it in the [frontend package.json](https://github.com/Seneca-CDOT/telescope/blob/master/src/frontend/package.json#L6).

Reply:
Removed.